schema/Figure.schema.yaml

```diff
@@ -0,0 +1,28 @@
+title: Figure
+'@id': stencila:Figure
+extends: CreativeWork
+role: primary
+status: experimental
+category: prose
+description: Encapsulates one or more images, videos, tables, etc, and provides captions and labels for them.
+properties:
+  label:
+    '@id': stencila:label
+    description: A short label for the figure.
+    type: string
+  caption:
+    '@id': schema:caption
+    description: A caption to display for the figure.
+    type: array
+    items:
+      $ref: InlineContent
+  object:
+    '@id': stencila:object
+    description: The actual content of the figure, usually an image, video or table, but can be any CreativeWork.
+    $ref: CreativeWorkTypes
+  alternatives:
```

Comment:
Instead of having `alternatives` and `object` what about having one of them? Personally, I think I lean towards just having `alternatives`. Often it would just have one `CreativeWork` in it, and where is had more than one, codecs may choose to just to encode the first one.

Reply:
Could both these comments be combined, use `content` for the media object(s) and retain `caption`?

Reply:
Yep, that sounds like a good approach - the alternative media object(s) could be simply be items in the `content` array. It might be best to restrict the type of items in `Figure.content` to be `anyof` `ImageObject`, `VideoObject`, `CodeChunk` (for reproducible figures) etc

Reply:
I tried to restrict the types like this but building the TS failed because the property was incompatible with the parent, so I just removed the content property off Figure and it will just inherit.